src/Components/Home.js

```diff
@@ -1,31 +1,152 @@
 import { Box, Typography } from "@mui/material";
 import React from "react";
+import { useState, useEffect } from "react";
+import cd1 from "../Assets/cd1.png";
+import cd2 from "../Assets/cd2.png";
+import cd3 from "../Assets/cd3.png";
+import S1 from "../Assets/S1.jpg";
+import p1 from "../Assets/p1.jpg";
+import p2 from "../Assets/p2.jpg";
+import CardSlider from "../Shared/CardSlider.js";
+import Testimonials from "../Shared/Testimonials.js";
+import { Swiper, SwiperSlide } from "swiper/react";
+import { Navigation, Pagination, Scrollbar, A11y } from "swiper/modules";
+import "swiper/css/navigation";
+import "swiper/css/pagination";
+import "swiper/css/scrollbar";
+import "swiper/css";
+import { EffectFade } from "swiper/modules";
 
+import Slider from "./Slider";
+import Content from "./Content";
 import "../Styles/Home.css";
+import SimpleCards from "../Shared/SimpleCards.js";
 const Home = () => {
+  const cardData = [
+    {
+      title: "Creative Plan & Design",
+      icon: <img src={cd1} alt="" className="rounded-xl w-32 h-32 ml-10" />,
+      description:
+        "Unleash the power of creativity and watch your projects come to life with precision and style. ",
+    },
+    {
+      title: "Restoration",
+      icon: <img src={cd2} alt="" className="rounded-xl w-32 h-32 ml-10" />,
+      description:
+        "Discover our restoration services that bring new life to old structures, We specialize in preserving the past while invorporating modern techniques for lasting results.",
+    },
+    {
+      title: " Inspection & Testing",
+      icon: <img src={cd3} alt="" className="rounded-xl w-32 h-32 ml-10" />,
+      description:
+        " Ensure the structural integrity of your building with our thorough inspection and testing services. Our experts provide detailed assessments to meet safety standards.",
+    },
+  ];
+  const cardData1 = [
```

Comment:
These are the constants, not changing at all so they need to be in a separate folder . under src>constants>index.js file and then export those constants
I have provided a link of my repo for better understanding how to import and export these constants 
@khadijatahira2001 
[https://github.com/Adda1321/Travel-webapp-landingpage/blob/master/constants/index.ts](url)